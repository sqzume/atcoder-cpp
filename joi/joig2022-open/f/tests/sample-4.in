9 11 5 10
1 2 1
1 3 2
2 3 2
2 9 2
3 9 1
4 9 1
8 9 1
5 8 1
5 7 1
4 7 2
6 7 2
2
6
7
8
9
