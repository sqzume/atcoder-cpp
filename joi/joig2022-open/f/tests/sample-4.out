2
Large
7
5
3
